tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -988,9 +988,19 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
                               default=())
     # TODO: isn't it a weights precision type
     common_group.add_argument('--data_type',
-                              help=mo_convert_params['data_type'].description,
+                              help='[DEPRECATED] Data type for model weights and biases. '
+                                   'If original model is in FP32 and --data_type=FP16 is specified, '
+                                   'model weights and biases are compressed to FP16, except for const values which '
```

Comment:
Two of the FAQ records say that we still can have truncations for very small or very big values. Where is the truth?

Reply:
@pavel-esir, please comment on the current sate and how it is going to be changed after you PR for FP16 constant compression is merged?